MSYS_install/install.ps1

```diff
@@ -0,0 +1,62 @@
+# https://www.powershellgallery.com/packages/AppVeyorBYOC/1.0.107-beta/Content/scripts%5CWindows%5Cinstall_msys64.ps1
+
+Write-Host "Installing MSYS2..." -ForegroundColor Cyan
+
+Set-Content -Value "Write-Host 'Sleep then kill gpg-agent.exe'; sleep 300; Stop-Process -name gpg-agent -Force" -Path .\kill-gpg-agent.ps1
+Start-Process powershell.exe -ArgumentList .\kill-gpg-agent.ps1
+
+if(Test-path C:\msys64) {
+    del C:\msys64 -Recurse -Force
+}
+
+# download installer
+$zipPath = "$($env:TEMP)\msys2-x86_64-latest.tar.xz"
+$tarPath = "$($env:TEMP)\msys2-x86_64-latest.tar"
+Write-Host "Downloading MSYS installation package..."
+(New-Object Net.WebClient).DownloadFile('http://repo.msys2.org/distrib/msys2-x86_64-latest.tar.xz', $zipPath)
+
+Write-Host "Untaring installation package..."
+7z x $zipPath -y -o"$env:TEMP" | Out-Null
+
+Write-Host "Unzipping installation package..."
+7z x $tarPath -y -oC:\ | Out-Null
+del $zipPath
+del $tarPath
+
+function bash($command) {
+    Write-Host $command -NoNewline
+    cmd /c start /wait C:\msys64\usr\bin\sh.exe --login -c $command
+    Write-Host " - OK" -ForegroundColor Green
+}
+
+[Environment]::SetEnvironmentVariable("MSYS2_PATH_TYPE", "inherit", "Machine")
+
+# install latest pacman
+#bash 'pacman -Sy --noconfirm pacman pacman-mirrors'
```

Comment:
Can we remove the lines with code that are commented? The same is happening below, I'm guessing these were experiments from Constantine?